shared/src/common/types/interview/interviews-update-request-dto.type.ts

```diff
@@ -1,7 +1,7 @@
 import { InterviewStatus } from '~/common/enums/enums';
 
 type InterviewsUpdateRequestDto = {
-  interviewerUserId: number | null;
+  interviewerUserId?: number | null;
```

Comment:
Just want to clarify if validation would handle this field absence correctly

Reply:
Yes, everything is okay, if interviewUserId will be absent
<img width="640" alt="image" src="https://user-images.githubusercontent.com/56650388/189618474-e59d8fbb-712a-491f-bf09-cc0150cc56a0.png">

Reply:
we want to avoid an optional keys